packages/frontend/src/components/Routing.js

```diff
@@ -292,6 +297,16 @@ class Routing extends Component {
         this.pollTokenFiatValue = null;
     };
 
+    // todo refactor: move to state manager
```

Comment:
This feels like something we'd never really get back to doing? Maybe we do it now or just remove it since this flow is only valid for the migration and will be removed afterwards.